src/app/modules/time-entries/pages/time-entries.component.spec.ts

```diff
@@ -442,4 +442,11 @@ describe('TimeEntriesComponent', () => {
     component.newEntry();
     expect(component.entry).toEqual(newEntry);
   });
+
+  it('When I stop dragging the modal, it should call dragEnded', () => {
+    const dragEndEvent = new DragEvent('CdkDragEnd');
+    spyOn(component, 'dragEnded');
+    component.dragEnded(dragEndEvent);
+    expect(component.dragEnded).toHaveBeenCalledWith(dragEndEvent);
+  });
```

Comment:
What about checking the position of the draggable(before and after call the dragEnded function). It was possible or not?

Reply:
Right now I am trying to verify the draggable positions and create a new unit test if it is the case.

Reply:
Update the unit test to verify that the reset method is being called